pkg/rtc/room.go

```diff
@@ -898,6 +898,7 @@ func (r *Room) onTrackUnpublished(p types.LocalParticipant, track types.MediaTra
 }
 
 func (r *Room) onParticipantUpdate(p types.LocalParticipant) {
+	r.protoProxy.MarkDirty(false)
```

Comment:
Adding this as this gets called from `SetPermissions` which could change `isPublisher` bool.

Reply:
if this is here, I think you can remove my previous changes when the track is published/unpublished.

Reply:
Cool, addressed in https://github.com/livekit/livekit/pull/1880/commits/a46fd85bbefcda794214309a02f3133c0fb7ad87